.project.qmlproject

```diff
@@ -6,23 +6,23 @@ Project {
     // List of module and plugin directories passed to QML runtime
     importPaths: [
         "easyDiffractionApp",
-        "../easyAppGui"
+        "../easyApp",

```

Comment:
merging `easyAppGui` and `easyAppLogic` into `easyApp` should probably be a separate PR so the extent of changes can be easily noticed. Piggybacking this large change onto the `updater` branch is not optimal.

Regarding the merge: are you convinced this is required? What if an app or script wants to use only the logic part? Currently it will have to import the whole module with lots of unused and unnecessary GUI stuff.
Are there enough interdependencies between eA Gui and eA Logic to warrant the merge?